deploy/requirements.yml

```diff
@@ -2,4 +2,4 @@
 # Install roles with
```

Comment:
Not necessarily, but I would keep it so we can easily update roles (although there's only one now).